docs/content/preview/migrate/yb-voyager-cli.md

```diff
@@ -113,7 +107,29 @@ yb-voyager export data --export-dir /path/to/yb/export/dir \
         --source-db-user username \
         --source-db-password password \
         --source-db-name dbname \
-        --source-db-schema schemaName #Include schema name only for Oracle migrations.
+        --source-db-schema schemaName \ #Not applicable for MySQL.
+        --oracle-db-sid string \ #Applicable only for Oracle
+        --oracle-home string \ #Applicable only for Oracle
+        --table-list string \
+        --exclude-table-list string
+```
+
+### export data status
+
+Get the status report of an ongoing or completed data export operation.
+
+#### Syntax
+
+```sh
+yb-voyager export data status [ <arguments> ... ]
+```
+
+- *arguments*: See [Arguments](#arguments)
+
+#### Example
+
+```sh
+yb-voyager export data status --export-dir /path/to/yb/export/dir
 ```
 
 ### import schema
```

Comment:
The new description of import schema should read:

During a typical migration workflow, the user is required to run the `import schema` command twice: first **without** the `post-import-data` flag and second time **with** the flag.
The second invocation **must** be done **after** the **data** import is already complete. The second invocation creates indexes and triggers in the target schema.